packages/web-runtime/src/components/SidebarNav/SidebarNav.vue

```diff
@@ -11,6 +11,7 @@
       :class="toggleSidebarButtonClass"
       class="toggle-sidebar-button oc-pb-s oc-pt-m"
       :aria-label="$gettext('Toggle sidebar')"
+      :aria-expanded="!closed"
```

Comment:
This needs to be a string. Otherwise a `falsy` value might cause the `aria-expanded` attribute to be removed from the html dom.

Reply:
![image](https://github.com/user-attachments/assets/2d1c402f-538b-4360-9495-592ea52eff1e)
Seems like vue is properly taking care of that